src/index.js

```diff
@@ -9,36 +9,39 @@ import generators from './generators';
 program
   .version(version)
   .description('Generate a CRUD application built with React, Redux and React Router from an Hydra-enabled API')
-  .usage('apiEntrypoint outputDirectory')
+  .usage('entrypoint outputDirectory')
   .option('-r, --resource [resourceName]', 'Generate CRUD for the given resource')
   .option('-p, --hydra-prefix [hydraPrefix]', 'The hydra prefix used by the API', 'hydra:')
   .option('-g, --generator [generator]', 'The generator to use, one of "react", "angular" etc.', 'react')
   .parse(process.argv);
 
-if (2 !== program.args.length) {
+if (2 !== program.args.length && (!process.env.API_PLATFORM_CLIENT_GENERATOR_ENTRYPOINT || !process.env.API_PLATFORM_CLIENT_GENERATOR_OUTPUT)) {
   program.help();
 }
 
-const generator = generators(program.generator)(program.hydraPrefix)
+const entrypoint = program.args[0] ? program.args[0] : process.env.API_PLATFORM_CLIENT_GENERATOR_ENTRYPOINT;
+const outputDirectory = program.args[1] ? program.args[1] : process.env.API_PLATFORM_CLIENT_GENERATOR_OUTPUT;
```

Comment:
why not `const entrypoint = program.args[0] || process.env.API_PLATFORM_CLIENT_GENERATOR_ENTRYPOINT`?

Reply:
indeed